### PR TITLE
feature/816 - add reverse proxy

### PR DESCRIPTION
**Pull request by dheitzer**
Added nginx reverse proxy as part of the chosen approach for [ticket 816](https://github.com/fecgov/fecfile-web-api/issues/816)

This ticket also requires a network policy to be put in place as part of deployment.

### CODE REVIEW

File: bin/cf_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  command="push --strategy rolling"
+else
+  command="push"
+fi
+
+# Deploy app
+cf ${command} ${app} -f ${manifest}
+
+# Add network policies
+#cf add-network-policy proxy cms
```
Comment (lbeaufort) [inline bin/cf_deploy.sh:63]: We'll need to add network policies in the deploy script.

File: manifest_dev.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+applications:
+  - name: fecfile-api-proxy
+    instances: 1
+    memory: 256M
+    disk_quota: 700M
+    routes:
+      - route: fecfile-web-api-dev.app.cloud.gov
```
Comment (lbeaufort) [inline manifest_dev.yml:8]: Let's make the route fecfile-api-proxy

File: bin/cf_deploy.sh (hunk 1 above)
Comment (dheitzer) [reply bin/cf_deploy.sh]: added to deploy script

File: manifest_dev.yml (hunk 2 above)
Comment (dheitzer) [reply manifest_dev.yml]: updated this and the other manifest with suggested URIs

Comment (lbeaufort) [reply manifest_dev.yml:8]: Since we separated the custom domain work into its own workstream, we'll add the *.fecfile.fec.gov routes with a later PR

**Review by lbeaufort — APPROVED** [review]
🎉 great work @dheitzer!

**Pull request merged**